scss/bootstrap/overrides/style/_wizard.scss

```diff
@@ -0,0 +1,114 @@
+.crmb-wizard {
+  .crmb-wizard-title,
+  .crmb-wizard-body {
+    margin-bottom: 30px;
+  }
+
+  .crmb-wizard-title .panel-body {
+    padding: 0;
+  }
+
+  .crmb-wizard-title .nav-pills {
+    padding: 0;
+
+    a,
+    a:active,
+    a:hover {
+      padding: 15px 20px;
+      color: $brand-primary;
+    }
+
+    li.active a {
+      background: $crm-light-gray;
+      position: relative;
+      color: $brand-primary;
+
+      &:after {
+        content: "";
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        right: -8px;
+        width: 0;
+        height: 100%;
+        border-color: transparent;
+        border-left-color: $crm-light-gray;
+        border-style: solid;
+        border-width: 30px 0px 30px 8px;
+      }
+
+      &:before {
+        content: "";
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        width: 0;
+        height: 100%;
+        border-color: transparent;
+        border-left-color: $crm-white;
+        border-style: solid;
+        border-width: 30px 0px 30px 8px;
+      }
+    }
+
+    li.active:first-child a:before {
+      display: none;
+    }
+
+    li:not(.active) a {
+      color: $gray-darker;
+
+      &:after {
+        content: "";
+        position: absolute;
+        bottom: 0;
+        right: -9px;
+        width: 1px;
+        height: 31px;
```

Comment:
according to http://caniuse.com/#search=transform
we shod use transform with prefixes